docs/github/v2/03-test-runner.md

```diff
@@ -194,6 +215,18 @@ In this folder a folder will be created for every test mode.
 _**required:** `false`_
 _**default:** `artifacts`_
 
+### coverageOptions
+
+Options for configuring code coverage, this configures the `-coverageOptions`
+parameter described unity's CodeCoverage documentation:
+[Using Code Coverage in batchmode](https://docs.unity3d.com/Packages/com.unity.testtools.codecoverage@1.2/manual/CoverageBatchmode.html).
+
+By default, the coverage is configured to generate an HTML report with additional metrics.
+This should be fine for most projects, but the webpage report can be quite large for projects
+with hundreds or thousands of large files. to disable the webpage report, remove the `generateHtmlReport`
+option from the coverage options. It is also recommended to add path or assembly filters for
+larger projects to keep the size of the report smaller if possible.
+
```

Comment:
Please follow the pattern of other parameters, using 
```
_**required:** `foo`_
_**default:** `bar`_
```

Reply:
Great, I'll update the PR